event-image-map.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <style>
+        * {
```

Comment:
Move css to a seprate file and minify/uglify it on production. Is there any caching that web view android would do if yes then generate hash of the css file so that it is different when changes are made to it.